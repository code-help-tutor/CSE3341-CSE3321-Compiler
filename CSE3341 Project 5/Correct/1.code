procedure t1 is
	record y;
begin
	y := new record[1];
	y[0] := 10;
end
