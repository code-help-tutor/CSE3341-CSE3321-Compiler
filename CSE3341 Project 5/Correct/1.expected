gc:1
gc:0
